tests/vmi_configuration_test.go

```diff
@@ -144,7 +148,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 	Context("[rfe_id:897][crit:medium][vendor:cnv-qe@redhat.com][level:component]for CPU and memory limits should", func() {
 
 		It("[test_id:3110]lead to get the burstable QOS class assigned when limit and requests differ", func() {
-			vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+			vmi := libvmi.NewAlpine()
```

Comment:
Correct me if I am wrong but I find an inconsistency here. While for Cirros you are setting masquerade pod network, here you are leaving it to be bridge one.

Reply:
Yea, and I missed it as well.
I do not understand why non-network tests need to care about this at all.
Is leaving the default problematic? It changes a bit the logic, but if it is passes I think it is better not to specify anything.
(needs a special note in the commit)

Reply:
You are correct that the NewRandomVMIWithEphemeralDisk added masquerade interface, but I fail to see NewAlpine to add a bridge interface, I think the NewAlpine function doesn't add any.

I'm a bit hesitant to fix this inconsistency though. I suspect that the masquerade interface was used here only because `NewRandomVMIWithEphemeralDisk` happened to add it by default. If it's not needed, isn't it better to not have it?

Reply:
The bridge will be added as default because there is nothing set. I am curious why you choose to add it in one instance and not in others. I am not arguing it needs to be here but I would expect consistency at least across one commit.